Resources/Prototypes/_CM14/Actions/Xeno/xeno_offense_actions.yml

```diff
@@ -98,15 +98,14 @@
   name: Headbutt (10) # TODO CM14 proper plasma costs
   description: Charge in a direction and push back any enemies in your way.
   components:
-  - type: WorldTargetAction
+  - type: EntityTargetAction
```

Comment:
Would this not make the defender always undershoot if they try to headbutt a fleeing marine?

Reply:
I've tested this and I think it works quite well, the headbutt has a small margin that is enough to catch up to a marine
https://youtu.be/Yx6fHGWab_U

Reply:
Tested this too, seems to work reliably